tests/Yaapii.Atoms.Tests/Text/TextOfTest.cs

```diff
@@ -210,6 +210,31 @@ public void ReadsFloatWithNumberFormatIntoText()
                     "Can't read text with format from float");
         }
 
+        [Theory]
+        [InlineData(true, "True")]
+        [InlineData(false, "False")]
+        public void ReadsBoolIntoText(bool input, string expected)
+        {
+            Assert.True(
+                new TextOf(input).AsString() == expected,
+                "Can't read text from bool"
+            );
+        }
+
+        [Theory]
+        [InlineData(true, "True")]
+        [InlineData(false, "False")]
+        public void ReadsBoolIntoTextWithCultureInfo(bool input, string expected)
+        {
+            Assert.True(
+                new TextOf(
+                    input,
+                    new CultureInfo("en-US")
+                ).AsString() == expected,
```

Comment:
💅🏻 Is there any culture that would change the outcome?

Reply:
I am not aware of any but didn't investigate too much.

Reply:
I didn't investigate that either, but i think there is probably a reason why the bool.ToString method accepts a CultureInfo as argument.

Reply:
Ok, thanks :)